### PR TITLE
Added Chart Generation

### PR DESCRIPTION
**Pull request by yogeshmaheshwari-idfy**
Added Chart Generation Capabilities

### CODE REVIEW

File: prompt_generator.py [hunk 1]
```diff
@@ -58,12 +59,18 @@ def get_data_from_metabase(**kwargs):
 
 
 def get_title_and_card_id():
-    with open("./metabase_cards.json", 'r') as file:
+    with open("/Users/yogeshmaheshwari/Desktop/AutoQBR/aqbr/metabase_cards.json", 'r') as file:
```
Comment (anubhavkumar-idfy) [inline prompt_generator.py]: Relative path

File: prompt_generator.py [hunk 2]
```diff
@@ -100,4 +108,25 @@ def get_prompt(current_start_date, current_end_date, ou_id):
 I am including a sample Email below for you to refer. This is only for reference, you can alter the email structure if need be. 
 
 {sample_email}
-""".format(prompt_body=prompt_body, prompt_header=prompt_header, prompt_tail=prompt_tail, sample_email=sample_email)
+""".format(prompt_body=prompt_body, prompt_header=prompt_header, prompt_tail=prompt_tail, sample_email=sample_email),jsonl_data
+
+def chart_generator(data):
+    data = read_jsonl("/Users/yogeshmaheshwari/Desktop/AutoQBR/aqbr/data.jsonl")
```
Comment (anubhavkumar-idfy) [inline prompt_generator.py]: Do not read from local file. You need to fetch from metbase and read. See how it has been done previously.

File: prompt_generator.py [hunk 3]
```diff
@@ -100,4 +108,25 @@ def get_prompt(current_start_date, current_end_date, ou_id):
...
+    system_text = """
+    You are a data visaualization and analysis expert using chartJS version 2 and height is 800 and widht is 1000.
```
Comment (anubhavkumar-idfy) [inline prompt_generator.py]: Move prompt to a new file

**Review by anubhavkumar-idfy — CHANGES_REQUESTED** [review]

File: prompt_generator.py (hunk 1 above)
Comment (yogeshmaheshwari-idfy) [reply prompt_generator.py]: Done

File: prompt_generator.py (hunk 2 above)
Comment (yogeshmaheshwari-idfy) [reply prompt_generator.py]: Removed

**Pull request merged**